images/virt-launcher/werf.inc.yaml

```
@@ -296,7 +304,7 @@ shell:
     xorriso==1.5.6-alt1:p11+348905.100.1.3 \
     dosfstools==4.2-alt3:sisyphus+344264.100.1.1 \
     mtools==1:4.0.43-alt1:sisyphus+317197.100.1.1 \
-    qemu-img==8.2.4-alt1:p11+350544.100.2.1 \
+    qemu-img==9.0.2-alt2:p11+353258.200.3.2 \
```

```suggestion
    qemu-img==8.2.4-alt1:p11+350544.100.2.1 \
```

No need to rebuild this image, let's revert qemu-img versin to use cached image.